subprojects/packagefiles/taskflow/meson.build

```diff
@@ -0,0 +1,15 @@
+project('taskflow', 'cpp',
+    version: 'v2.7.0',
+    license: 'MIT',
+)
+
+# Header-only library
+taskflow_inc = include_directories('.')
```

Comment:
Thanks. I am not sure how it will apply to subprojects though. What is the use-case here? Could you please show me an example?

My use-case is to generate a release version of the standalone executable `my-app`, which will strip all debug symbols related to Taskflow. I will apply line-time optimization (LTO) to inline / eliminate all Taskflow internal mechanisms. Therefore, only `my-app` will ever get installed.

Reply:
Thanks. I am not sure what scenario requires an installation of header file in the subprojects. Could you show me?

My use-case is to build an standalone executable `my-app`. I generate the release version with link-time optimization (LTO), so all relevant symbols in `Taskflow` will be either inlined, optimized out, and striped. Only `my-app` will be installed.

Reply:
For compatibility with the upstream CMakeLists.txt basically. You can restrict from installing subprojects using `meson install --skip-subprojects` even if the subproject itself has installation wired up.

Reply:
Acknowledged. I pushed changes based on my very early understanding of the `pkgconfig` meson module. I am still figuring out how to generate taskflow pkgconfig that depends on `pthread`. Appreciate if you can polish my draft code.